src/main/telemetry/smartport.c

```diff
@@ -149,8 +151,8 @@ enum
     FSSP_DATAID_A4         = 0x0910
 };
 
-// if adding more sensors then increase this value
-#define MAX_DATAIDS 17
+// if adding more sensors then increase this value (should be equal to the maximum number of ADD_SENSOR calls)
+#define MAX_DATAIDS 20
```

Comment:
+3 because it should have been +1 when core temp sensor (FSSP_DATAID_T11) was added, and +2 for this PR.